src/lib/RpcApi.ts

```diff
@@ -147,17 +147,22 @@ export default class RpcApi {
                     hasRequest: !!this._staticStore.request,
                 });
 
+                let account;
+                if (request && 'walletId' in request) {
+                    account = await WalletStore.Instance.get((request as ParsedSimpleRequest).walletId);
+                    if (!account && requestType !== RequestType.LOGOUT) {
+                        // no account found although it's required
+                        state.reply(ResponseStatus.ERROR, new Error('Account ID not found'));
```

Comment:
This should probably use `this.reject(new Error(...))` instead.

Reply:
Shouldn't it then be changed here
https://github.com/nimiq/accounts/blob/ddae45bd6b1c27165c9ceb22841c8ebdc455b158/src/lib/RpcApi.ts#L141
and here
https://github.com/nimiq/accounts/blob/ddae45bd6b1c27165c9ceb22841c8ebdc455b158/src/lib/RpcApi.ts#L227
too?

Reply:
Yes I guess it can. Those were forgotten when we added the reject method.